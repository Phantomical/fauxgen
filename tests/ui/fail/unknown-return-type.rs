use fauxgen::*;

#[generator]
fn some_generator() -> Undef {
    todo!()
}

fn main() {}
