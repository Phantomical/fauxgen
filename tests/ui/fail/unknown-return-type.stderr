error[E0412]: cannot find type `Undef` in this scope
 --> tests/ui/fail/unknown-return-type.rs:4:24
  |
4 | fn some_generator() -> Undef {
  |                        ^^^^^ not found in this scope
  |
help: you might be missing a type parameter
  |
4 | fn some_generator<Undef>() -> Undef {
  |                  +++++++
